### ISSUE TITLE
Does not seem to work even with sample code provided

### ISSUE THREAD

**Issue by brutaldev** [Fody/ConfigureAwait#430]
I am not a Patron,  but the fact that this does not work at all would probably prevent anyone (including myself) from ever becoming one in the first place...

#### Describe the issue

Noticed that the `.ConfigureAwait(false)` was not being weaved into projects using the `<ConfigureAwait ContinueOnCapturedContext="false" />` configuration option, I tried all other available options and none of them successfully weave the code when checking with dotPeak.  The sample project I created used the XML config on the `.csproj`, a `FodyWeavers.xml` file, class level attribute and assembly level attribute and even with all of them in place the code remains unchanged with no `.ConfigureAwait(false)` weaved in.

![image](https://user-images.githubusercontent.com/1929531/160922367-54f19479-50bb-4469-8a94-eec6dd330a26.png)

#### Minimal Repro

Created a simple project with the [sample code provided](https://github.com/Fody/ConfigureAwait#example) to prove that no available options work: https://brutaldev.com/download/FodyTest.zip

Compile the project and inspect the assemblies, there is no weaving of the example code and it is not as expected from the documentation.

#### Make an effort to fix the bug

No idea where to even start because none of the options work so this is less of a "bug" and more of a complete failure of the add-in to work at the most basic level.

**Comment by xaevik** [comment]
@brutaldev I would check those compiled artifacts with ILSpy. The problem with JetBrains dotPeek is that if it can find a reference to the original source files it will display those instead of the decompiled IL.

We have numerous .NET 6 class libraries and projects using ConfigureAwait.Fody and it does attach .ConfigureAwait(false).

**Comment by brutaldev** [comment]
@xaevik I did verify with multiple decompilers including ILSpy. I provided the sample code and project in it's most basic form using the example code which works in older versions of .NET but not on .NET6 projects as previously reported by @LukyanovOleg. Even the unit tests fail when using .NET 6.

**Comment by xaevik** [comment]
@brutaldev So I went ahead and grabbed your sample project and compiled it. I am showing that the ConfigureAwait method call is being weaved:

![image](https://user-images.githubusercontent.com/62903002/186250083-d9008df6-781a-4e84-9eff-580981ae215c.png)

I tested this against **Visual Studio 2022 17.3.0** on **Windows 10 21H2 Enterprise** against **.NET 6 6.0.8 (6.0.400)**

**Comment by brutaldev** [comment]
@xaevik Well I certainly stand corrected.

I needed to update ILSpy to 8/7.x to see what you are seeing and dotPeek at the latest version still does not indicate the `ConfigureAwait` call being made. So this does in fact work as expected and I apologize for wasting any time on this.

I have verified this working on a much larger project as well and confident now the weaving is working as expected.

Issue #391 is probably a duplicate of this and should probably be closed as well.
